guides/common/modules/proc_deploying-a-virt-who-configuration-on-target-server.adoc

```diff
@@ -0,0 +1,70 @@
+[id="deploying-a-virt-who-configuration-on-target-server_{context}"]
+= Deploying a virt-who configuration on {a-target}
+
+You can deploy a virt-who configuration for {provider} on {a-target}.
+ifdef::vmware-el,nutanix-el,hyperv-el[]
+The server can be physical or virtual.
+endif::[]
+
+Global configuration values apply to all virt-who configurations on this server and are overwritten if you deploy a new virt-who configuration.
+
+ifdef::kvm-hypervisor,openstack-hypervisor,kubevirt-hypervisor[]
+.Prerequisites
+* You have registered the hypervisor to {ProjectServer}.
+endif::[]
+ifdef::vmware-el,nutanix-el,hyperv-el[]
+.Prerequisites
+* You have registered {the-target} to {ProjectServer}. For more information, see {ProvisioningDocURL}registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts[Registering hosts by using global registration] in _{ProvisioningDocTitle}_.
+endif::[]
+
+.Procedure
+ifdef::vmware-el,nutanix-el,hyperv-el[]
+. Log in to {the-target}.
+. Add a TCP port for communication between virt-who and {ProjectServer} to the runtime configuration:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ firewall-cmd --add-port="{port}/tcp"
+----
+
+. Add a TCP port for communication between virt-who and {ProjectServer} to the permanent configuration:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+$ firewall-cmd --add-port="{port}/tcp" --permanent
```

Comment:
question (non-blocking): unsure if line 27 is necessary if you run line 34. Maybe there's a command that does both at the same time?

Reply:
I checked the [firewall-cmd man page](https://firewalld.org/documentation/man-pages/firewall-cmd.html). Looks like it requires 2 separate commands:

> If you want to make a change in runtime and permanent configuration, use the same call with and without the --permanent option.